Week3/homework/2-step3.js

```diff
@@ -2,7 +2,34 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+  for (let i = startIndex; i <= stopIndex; i++) {
+    values.push(i);
+
+    if (i % 3 === 0) {
+      threeCallback(i);
+    }
+
+    if (i % 5 === 0) {
+      fiveCallback(i);
+    }
+
+    if (i % 3 === 0 && i % 5 === 0) {
+      console.log(i);
+    }
+
+  }
+
+  return values;
+
```

Comment:
Nicely done, and laid out.  Easy to read.  And you only make one run through the array.